asg-green.tf

```diff
@@ -0,0 +1,45 @@
+data "aws_ami" "nginx-green" {
+  most_recent = true
+  owners = ["699330879220"]
+
+  filter {
+    name   = "root-device-type"
+    values = ["ebs"]
+  }
+
+  filter {
+    name   = "virtualization-type"
+    values = ["hvm"]
+  }
+
+  filter {
+    name   = "image-id"
+    values = ["ami-02a0ce8867b366954"]
```

Comment:
Same here, no image id but use the image name instead.

Reply:
ok Pascal.. I will correct the code as per your suggestions...